projects/demo/src/modules/app/home/home.template.html

```diff
@@ -111,6 +111,17 @@
                         <code>npm i @taiga-ui/styles</code>
                         and put this code:
 
+                        <tui-notification
+                            status="warning"
+                            class="tui-space_top-3"
+                        >
+                            [WARNING]: In version 3.0 we removed support normalize(v7) out-of-the-box
+                            <br />
+                            You need to manually download css file from https://necolas.github.io/normalize.css/
```

Comment:
Something like this:
```suggestion
                            You need to manually <a tuiLink href="https://necolas.github.io/normalize.css/">download css file</a>
```

Reply:
fixed